applications/jonny5/src/j5_limits.erl

```diff
@@ -64,10 +67,12 @@
 -spec get(ne_binary()) -> limits().
 get(Account) ->
     AccountId = wh_util:format_account_id(Account, 'raw'),
-    case wh_cache:peek_local(?JONNY5_CACHE, ?LIMITS_KEY(AccountId)) of
-        {'ok', Limits} -> Limits;
+    Limits = case wh_cache:peek_local(?JONNY5_CACHE, ?LIMITS_KEY(AccountId)) of
+        {'ok', _Limits} -> _Limits;
```

Comment:
Using the underscore-prefixed variable name is bad practice (other than when logging). Maybe just use `{'ok', Lims} -> Lims;` or something similarly ghetto :). Using `_Limits` reads as "This Variable contains the limits but I don't care about it here and it should be ignored", which isn't the case.